opentreemap/treemap/models.py

```diff
@@ -507,6 +507,7 @@ class InstanceUser(Auditable, models.Model):
     role = models.ForeignKey(Role)
     reputation = models.IntegerField(default=0)
     admin = models.BooleanField(default=False)
+    last_seen = models.DateField(null=True, blank=True)
```

Comment:
I saw that we are using `now().date()`, so we would only generate an audit row once per day. While this is a lot better, I think those rows are mostly just adding noise.